### PR TITLE
creating pr

### PR DESCRIPTION
**Pull request by ShubhamGupta24**
Good Comment

### CODE REVIEW

**Review by ShubhamGupta24 — COMMENTED** [review]
ok

File: server.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// Hi blogApp developer
```
Comment (agrVidushi) [inline server.js:1]: remove console from blogValidator

Comment (agrVidushi) [reply server.js:1]: Always try and seggregate code into controller and service wherein service contains your business logic

Comment (agrVidushi) [reply server.js:1]: remove unnecessary console from user-model file

Comment (agrVidushi) [reply server.js:1]: remove unnecessary comments from validate-middleware

**Pull request merged**